launcher/launcher.go

```diff
@@ -558,6 +566,17 @@ func (launcher *Launcher) doStopAction(instance *instanceInfo) {
 	})
 }
 
+func (launcher *Launcher) releaseRuntime(instance *instanceInfo, service *serviceInfo) (err error) {
+	if service == nil || service.serviceConfig.Permissions != nil {
+		if registerErr := launcher.instanceRegistrar.UnregisterInstance(
```

Comment:
you could could use only one err

Reply:
Currently here is only one condition but more are added in the following commits.